recipes/snzip/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set author = "kubo" %}
+{% set name = "snzip" %}
+{% set version = "1.0.4" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "a45081354715d48ed31899508ebed04a41d4b4a91dca37b79fc3b8ee0c02e25e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://bintray.com/{{ author }}/generic/download_file?file_path={{ name }}-{{ version }}.tar.gz
```

Comment:
One difference is that the distribution release have already the configuration file. Otherwise we need to run `autogen.sh`.

Would you prefer github snapshots?

In the case of #2348, the distribution release contains a snapshot of the plugins which are tracked via git submodules. When I tried to build from the github snapshot I found that some submodules references are missing, thus failing to update the plugins' libraries.